File: levintoo/reverse-integer.php
<?php
/*
 * Write a program that takes an integer as input and return an integer with reversed digit ordering.
 * 
 * Example inputs and expected outputs:
 * Input: 500  -> Output: 5
 * Input: -56  -> Output: -65
 * Input: -90  -> Output: -9
 * Input: 91   -> Output: 19
 */

function reverseInteger(int $num): int {
    $isNegative = $num < 0;

    $reversedString = strrev((string) abs($num));

    $reversedNumber = (int) $reversedString;

    return $isNegative ? -$reversedNumber : $reversedNumber;
}

// Test cases
echo reverseInteger(500) . "\n"; // Output: 5
echo reverseInteger(-56) . "\n"; // Output: -65
echo reverseInteger(-90) . "\n"; // Output: -9
echo reverseInteger(91) . "\n"; // Output: 19

?>

File: NewtonMutugi/reverse_integer.py
def reverse_integer(num: int) -> int:
    """
    Takes an integer as input and
    returns an integer with reversed digit ordering.
    """
    sign = -1 if num < 0 else 1
    reversed_num = int(str(abs(num))[::-1])
    return sign * reversed_num

if __name__ == "__main__":
    # Test for positive integer
    print(reverse_integer(234))
    # Test for negative integer
    print(reverse_integer(-234))
